src/devices/QwiicButton/QwiicButton.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Target framework should be: `<TargetFrameworks>net5.0;netcoreapp2.1</TargetFrameworks>`

Reply:
Fixed